developer/js/source/index.ts

```diff
@@ -251,7 +257,7 @@ export default class LexicalModelCompiler {
           createTrieDataStructure(filenames, modelSource.searchTermToKey)
         }, {\n`;
         if (wordBreakingSource) {
-          func += `  wordBreaking: ${wordBreakingSource},\n`;
+          func += `  wordBreaker: ${wordBreakingSource},\n`;
```

Comment:
Any reason why we use `wordBreaking` in the source and `wordBreaker` in the compiled code? Seems like a potentially confusing name change.

Reply:
My preference is `wordBreaker` - we're specifying a function that breaks words.  Same part of speech as the rest:  `format`, `sources`, etc - so it's more grammatically consistent with everything else.

`wordBreaking` is technically a gerund, not a noun.  It does function like a noun, but is a little more grammatically complex and might be less intuitive for non-native speakers as a result.